### PR TITLE
My Body Is Ready

### PR DESCRIPTION
**Pull request by csgamble**

### CODE REVIEW

**Comment by ktarplee** [comment]
Looks like the tests are failing.

**Comment by csgamble** [comment]
Yeah, I wasn’t sure how to check at first, so I just went for it. I’ll fix that.

**Comment by ktarplee** [comment]
Still failing man.

File: csgamble/test.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+__author__ = 'Cam-Mac'
+"""Test file for Magic 1089"""
+
+
+from . import Magic
+
+
+# creates a list of numbers 100-999
+container = range(100, 999)
```
Comment (ktarplee) [inline csgamble/test.py]: why is this global?

**Pull request merged**